packages/cli/package.json

```diff
@@ -0,0 +1,51 @@
+{
+  "name": "@iov/cli",
+  "version": "0.0.3",
+  "description": "Command line interface",
+  "contributors": ["IOV SAS <admin@iov.one>", "Simon Warta"],
+  "license": "Apache-2.0",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/confio/cosmwasm-js/tree/master/packages/cli"
+  },
+  "publishConfig": {
+    "access": "public"
+  },
+  "scripts": {
+    "format": "prettier --write --loglevel warn \"./src/**/*.ts\"",
+    "format-text": "prettier --write --prose-wrap always --print-width 80 \"./*.md\"",
+    "lint": "eslint --max-warnings 0 \"**/*.{js,ts}\" && tslint -t verbose --project .",
+    "build": "tsc",
+    "build-or-skip": "[ -n \"$SKIP_BUILD\" ] || yarn build",
+    "selftest": "yarn build-or-skip && ./bin/cosmwasm-cli --selftest",
+    "test-node": "node jasmine-testrunner.js",
+    "test": "yarn build-or-skip && yarn test-node"
+  },
+  "bin": {
+    "cosmwasm-cli": "bin/cosmwasm-cli"
+  },
+  "files": [
+    "build/",
+    "types/",
+    "tsconfig_repl.json",
+    "*.md",
+    "!*.spec.*",
+    "!**/testdata/"
+  ],
+  "dependencies": {
+    "@cosmwasm/sdk": "^0.0.3",
```

Comment:
Interesting, just our sdk lib, no bcp. I think this is a good idea.

However, I think the current sdk is a bit low-level sometimes (more or less a clean wrapper around the RestServer without too many deps). I would suggest adding (not this PR but soon) another package with a bit higher level helpers around sdk - adding imports for key management and signing transactions, also constructing messages.

This is another issue, but yeah my main thought here

Reply:
One thing that really bothers me is that we can't do any kind of signing without BCP types. I imported all the BCP stuff now and added an example from the rest client tests.